[ci] Push Performance CI build error

## Summary

```
   Compiling h2 v0.3.4
   Compiling tower v0.4.8
   Compiling opentelemetry v0.16.0
   Compiling trust-dns-resolver v0.20.3
   Compiling comfy-table v4.1.1
error: failed to run custom build command for `common-store-api-sdk v0.1.0 (/home/runner/work/databend/databend/common/store-api-sdk)`

Caused by:
  process didn't exit successfully: `/home/runner/work/databend/databend/target/release/build/common-store-api-sdk-cc2ef98311161ab3/build-script-build` (exit status: 1)
  --- stdout
  cargo:rerun-if-changed=/home/runner/work/databend/databend/common/store-api-sdk/proto/queryflight.proto
  cargo:rerun-if-changed=/home/runner/work/databend/databend/common/store-api-sdk/proto/storeflight.proto

  --- stderr
  error: 'rustfmt' is not installed for the toolchain 'nightly-2021-09-11-x86_64-unknown-linux-gnu'
  To install, run `rustup component add rustfmt --toolchain nightly-2021-09-11-x86_64-unknown-linux-gnu`
warning: build failed, waiting for other jobs to finish...
error: build failed
Error: Process completed with exit code 101.
```

https://github.com/datafuselabs/databend/runs/3660055657

--- a/.github/workflows/databend-perf-docker.yml
+++ b/.github/workflows/databend-perf-docker.yml
@@ -21,6 +21,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Rust setup
+        run: |
+          bash ./scripts/setup/dev_setup.sh
+          cargo install --version 0.1.16 cross
+
       - name: Build Perf Tool
         run: |
           if [ ${{ matrix.config.cross }} = true ]; then

